filter.py

```diff
@@ -39,7 +39,7 @@ def process_markdown(metadata, body, contributors):
     authors = []
     authors_xmp = []
     affiliations = {}
-    acknowledgements = []
+    acknowledgements = ["# Acknowledgements {#sec:acknowledgements}"]
```

Comment:
I think that an [unnumbered section](https://pandoc.org/chunkedhtml-demo/8.3-headings.html) would be better for the Acknowledgements:

```suggestion
    acknowledgements = ["# Acknowledgements {- #sec:acknowledgements}"]
```